Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook

```diff
@@ -144,73 +144,19 @@
                         },
                         "queryType": 0,
                         "resourceType": "microsoft.operationalinsights/workspaces",
-                        "value": ""
+                        "value": "7107BB20-910A-4F15-84F4-2864CF1E2099"
```

Comment:
Don't need the GUID, needs to be "value": ""

Reply:
Done